_i18n/en.yml

```diff
@@ -147,42 +147,50 @@ publications-tweet_feed:
+   subtitle: We will help you tailor a Continuous Delivery Pipeline so that your software is responsive to your needs. We are experts in technologies that are enablers for Continuous Delivery. We appreciate how the development team and business need to be aligned to realise the full potential of this approach.
 services-software_creation-ux_ui_design:
    title: UX & UI Design
-   subtitle: We have expertise in designing visual identity and user experiences and understand how this efforts fits best in an Agile environment. We will help you define the brand identity for your project and create UX and UI designs as part of your software development effort.
+   subtitle: We have expertise in designing visual identity and user experiences. We understand how this effort fits best in an Agile Softeware Deveopment environment. We will help you build the brand identity and create UX and UI designs as part of your software development effort.
 services-expert_advice:
    title: Expert Advice
    subtitle: <p><strong>You can draw from our expertise to fast track your initiatives.</strong></p>
            <p>We can help you start a new project, instil XP practices or Devops culture into your existing teams, plan your architectural evolution or adopt a new technology.</p>
            <p>We are available to run workshops and engage with you on a short-term basis to ensure that you put your best foot forward for your next initiative.</p>
 services-expert_advice-enterprise_architecture:
    title: Enterprise Architecture
-   subtitle: <p>Our Craftsmen have a wealth of expertise and experiences in dealing with large scale systems in finance, media and government.</p>
-           <p>We will review your approach and help you make the most effective decisions for the future.</p>
+   subtitle: Our Craftsmen have a wealth of expertise and experiences in dealing with large scale systems in finance, media and government. We will review your approach and help you make the most effective decisions for the future.
 services-expert_advice-software_architecture_and_design:
    title: Software Architecture & Design
-   subtitle: <p>We will review your current architecture and help you create a risk-based view of your software solution. We can advise on the best way to evolve your architecture to reduce those risks.</p>
-           <p>If you are looking to evolve your system into microservices or create a microservices based architecture from scratch, we will help you understand the best approach according to your needs.</p>
+   subtitle: We will review and help create an apporach to evolve your architecture towards an agreed vision. We have deep knowledge and experience in cloud technologies and enterprise architectures, including Domain Driven Design, Microservices, Event Sourcing and Serverless.
```

Comment:
We will review and help create an approach to evolve your architecture towards an agreed vision. We have deep knowledge and experience in cloud technologies and enterprise architectures, including Domain Driven Design, Microservices, Event Sourcing and serverless architectures.